lib/shared/types.ts

```diff
@@ -48,6 +48,7 @@ export type CircuitWebWorker = {
   execute: (code: string) => Promise<void>
   executeWithFsMap: (opts: {
     entrypoint?: string
+    mainComponentPath?: string
     fsMap: Record<string, string>
```

Comment:
this api is fine, also consider adding `mainComponentExport?: string` (but we also guess the export by default)